src/main/java/oc/wh40k/units/im/IMDeathCultAssassins.java

```diff
@@ -27,13 +33,26 @@ public void refreshen() {
         if (squad.getModelle() <= 2)
             power = 1;
         else if (squad.getModelle() <= 4)
-            power = 2;
-        else if (squad.getModelle() <= 6)
             power = 3;
-        else if (squad.getModelle() <= 8)
-            power = 4;
-        else if (squad.getModelle() <= 8)
+        else if (squad.getModelle() <= 6)
             power = 5;
+        else if (squad.getModelle() <= 8)
+            power = 6;
+        else if (squad.getModelle() <= 10)
+            power = 8;
+        
+        setEintragsCNT(getCountFromInformationVector("MINISTORUM_PRIEST") > 0 ? 0 : 1);
+        
+        if(getCountFromInformationVector("MINISTORUM_PRIEST") < 1 && getCountFromInformationVector("ECCLESIARCHY_BATTLE_CONCLAVE") > 1) {
+        	setFehlermeldung("MAX 1 CONCLAVE");
+        } else {
+        	setFehlermeldung("");
+        }
+    }
```

Comment:
This block exists in multiple places. Maybe create a common super class for Sororitas `Eintrag` with this functionality as a method?

Reply:
That's quite hard to do in a clean way with the current architecture. I think the cleanest way would be to let each `Eintrag` have a list of rules that apply to it and go through them on creation, refresh and deletion. But that should be discussed in another issue: #148